src/app/pages/pages.module.ts

```diff
@@ -2,10 +2,11 @@ import { NgModule } from '@angular/core';
 
 import { MainModule } from './main/main.module';
 import { SharedModule } from '../shared/shared.module';
+import { ReactiveFormsModule } from '@angular/forms';
 
 @NgModule({
   declarations: [],
-  imports: [SharedModule, MainModule],
+  imports: [SharedModule, MainModule, ReactiveFormsModule],
```

Comment:
```suggestion
  imports: [SharedModule, MainModule],
```

`ReactiveFormsModule` is already added in `SharedModule` :+1: